service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/request/EhrRequestHandler.java

```diff
@@ -0,0 +1,20 @@
+package uk.nhs.adaptors.gp2gp.ehr.request;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.adaptors.gp2gp.common.task.TaskDispatcher;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class EhrRequestHandler {
+
+    private final TaskDispatcher taskDispatcher;
+
+    public void handleRequest(String value) {
+        // FIXME: NIAD-776 create actual impl
```

Comment:
Do we really need this comment?

Reply:
FIXME's are ok tagged with a ticket number since that work is planned and scheduled. I don't like things like // FIXME: we should fix the some day

Reply:
I mean if we could just put this information in a jira story instead of putting it in a service code.

Reply:
removed